lib/api/src/lib.rs

```diff
@@ -446,11 +446,36 @@ mod value;
 pub mod vm;
 
 #[cfg(feature = "sys")]
-mod sys;
+/// sys
+pub mod sys;
 
 #[cfg(feature = "sys")]
 pub use sys::*;
```

Comment:
Do we want this?

Reply:
Well we're saying `use wasmer::sys::Artifact instead`, so this module should be available for users to be able to migrate.